File: Problem2/problem2.py
fibA = 1
fibB = 1
even_sum = 0

four_million = 4000000

while fibA <= four_million:
    if not fibA & 1:
        even_sum += fibA
    temp = fibA
    fibA += fibB
    fibB = temp

print(even_sum)

File: Problem2/Problem2.cpp
#include <iostream>

int main() {
    int fibA = 1, fibB = 1, sum = 0, temp;
    int FOUR_MILLION = 4000000;

    while (fibA <= FOUR_MILLION) {
        if ((fibA & 1) != 1) {
            sum += fibA;
        }
        temp = fibA;
        fibA += fibB;
        fibB = temp;
    }
    std::cout << sum << "\n";
}